src/directive.js

```diff
@@ -68,7 +57,8 @@
                         stoken: scope.stoken || attrs.stoken || null,
                         theme: scope.theme || attrs.theme || null,
                         tabindex: scope.tabindex || attrs.tabindex || null,
-                        size: scope.size || attrs.size || null
+                        size: scope.size || attrs.size || null,
+                        'expired-callback': expired
```

Comment:
I kinda think the arguments are redundant; as it immediately sets them on the object which is also passed as an argument. Might as well just put them on the object in the first place.... which is what I did in #108. If you hate it, I can change it back... reluctantly.